tests/CMakeLists.txt

```diff
@@ -141,13 +141,17 @@ if(QUDA_DIRAC_WILSON
    OR QUDA_DIRAC_STAGGERED)
   add_executable(blas_test blas_test.cpp)
   target_link_libraries(blas_test ${TEST_LIBS})
+  target_compile_options(blas_test PRIVATE $<$<COMPILE_LANG_AND_ID:CUDA,Clang>: --cuda-path=${CUDAToolkit_TARGET_DIR} --cuda-gpu-arch=${QUDA_GPU_ARCH}>)
+
   quda_checkbuildtest(blas_test QUDA_BUILD_ALL_TESTS)
   install(TARGETS blas_test ${QUDA_EXCLUDE_FROM_INSTALL} DESTINATION ${CMAKE_INSTALL_BINDIR})
 endif()
 
 if(QUDA_MULTIGRID)
-  add_executable(multigrid_benchmark_test multigrid_benchmark_test.cu)
+  add_executable(multigrid_benchmark_test multigrid_benchmark_test.cpp)
   target_link_libraries(multigrid_benchmark_test ${TEST_LIBS})
+  target_compile_options(multigrid_benchmark_test PRIVATE $<$<CUDA_COMPILER_ID:Clang>: --cuda-path=${CUDAToolkit_TARGET_DIR} --cuda-gpu-arch=${QUDA_GPU_ARCH}>)
+
```

Comment:
any luck with the `--cuda-path` flags? They should not be needed to compile a `.cpp`

Reply:
I have commented this out for now. We should check that it does not cause any issue and then remove the line.